__tests__/rpcProvider.test.ts

```diff
@@ -1,17 +1,61 @@
-import { RpcProvider } from '../src';
-import { describeIfRpc, getTestProvider } from './fixtures';
+import { isBN } from 'bn.js';
+
+import { Account, RpcProvider, ec } from '../src';
+import {
+  compiledErc20,
+  compiledOpenZeppelinAccount,
+  describeIfRpc,
+  getTestAccount,
+  getTestProvider,
+} from './fixtures';
 
 describeIfRpc('RPCProvider', () => {
-  let provider: RpcProvider;
+  const provider = getTestProvider() as RpcProvider;
+  const account = getTestAccount(provider);
+  let erc20Address: string;
+  let accountPublicKey: string;
 
   beforeAll(async () => {
-    provider = getTestProvider() as RpcProvider;
+    expect(account).toBeInstanceOf(Account);
+
+    const erc20Response = await provider.deployContract({
+      contract: compiledErc20,
+    });
+
+    erc20Address = erc20Response.contract_address;
+
+    await provider.waitForTransaction(erc20Response.transaction_hash);
+
+    const accountKeyPair = ec.genKeyPair();
+    accountPublicKey = ec.getStarkKey(accountKeyPair);
   });
 
   describe('RPC methods', () => {
     test('getChainId', async () => {
       const chainId = await provider.getChainId();
       expect(chainId).toBe('0x534e5f474f45524c49');
     });
+
+    test('deployContract', async () => {
+      const { contract_address, transaction_hash } = await provider.deployContract({
+        contract: compiledOpenZeppelinAccount,
+        constructorCalldata: [accountPublicKey],
+        addressSalt: accountPublicKey,
+      });
+      await provider.waitForTransaction(transaction_hash);
+      expect(contract_address).toBeTruthy();
+      expect(transaction_hash).toBeTruthy();
+    });
+
+    test('getEstimateFee', async () => {
```

Comment:
This test should actually be inside `account.tests.ts` the way it is written (and there already a similar test exists)... but I think the intent was to actually test the rpc `getEstimateFee` function ?